src/Hepteracts.ts

```diff
@@ -354,28 +378,35 @@ export const hepteractToOverfluxOrbDescription = () => {
  * @returns Alert of either purchase failure or success
  */
 export const tradeHepteractToOverfluxOrb = async () => {
-    const maxBuy = Math.floor(player.wowAbyssals / 250000)
-    const hepteractInput = await Prompt('How many Orbs would you like to purchase? You can buy up to ' + format(maxBuy, 0, true) +  ' with your hepteracts.')
+    const maxBuy = Math.floor(player.wowAbyssals / 250000);
+    const hepteractInput = await Prompt('How many Orbs would you like to purchase?\n You can buy up to ' + format(maxBuy, 0, true) +  ' with your hepteracts.');
     const toUse = Number(hepteractInput);
+    if (toUse === null) {
```

Comment:
actually this should just be removed, I mistook it for something else (NaN and 0 are checked below)

Reply:
to correct